internal/app/plan.go

```diff
@@ -133,6 +133,12 @@ func releaseWithHooks(cmd orderedCommand, wg *sync.WaitGroup, sem chan struct{},
 		wg.Done()
 		<-sem
 	}()
+	if cmd.targetRelease != nil && !flags.destroy {
+		err := fmt.Errorf("got an empty target release")
```

Comment:
This says that it is an empty target release but the condition is actually checking if it’s not nil. Is this a mistake or I am just not getting this part? :-)

Reply:
it's just the way I've worded it, I'll update it.

Reply:
the idea here is to fail faster, allowing it to go to `execOne` would have it run the first command but then return with an error...